packages/menu/package.json

```diff
@@ -0,0 +1,19 @@
+{
+   "name": "@ifixit/menu",
+   "version": "0.0.0",
+   "main": "./index.tsx",
+   "types": "./index.tsx",
+   "license": "MIT",
+   "exports": {
+      ".": "./index.ts"
+   },
+   "scripts": {
+      "build": "",
+      "clean": "",
+      "type-check": "tsc --pretty --noEmit"
+   },
+   "devDependencies": {
+      "@ifixit/tsconfig": "workspace:*",
+      "typescript": "4.8.4"
```

Comment:
I'm almost *certain* this package still wants react and chakra as some kind of dependencies (they are assumed and used in the code I think). So pnpm needs to make a symlink in order to access them. 

It's possible that it's still works because we are unintentionally hoisting everything: https://github.com/iFixit/ifixit/blob/master/common/config/rush/.npmrc#L48

Likely we only meant to hoist the listed things.

Reply:
I still don't believe we use react or chakra in this package. The only actual code is in menu.ts, which only consists of 50 lines of typescript types and an enum. Nothing is imported into that file either. 
The old UI package that menu.ts was in definitely had react and chakra, but the whole reason for extracting menu is so we don't need to rely on the extra dependencies of the UI package.

Reply:
Oh wow, I didn't notice how little was in it. You're right. Carry on then!